app/javascript/styles/components.scss

```diff
@@ -1281,6 +1281,10 @@
   }
 }
 
+.media-modal .react-swipeable-view-container {
+  align-items: center;
```

Comment:
Right, but it's superseded by `opacity: 0` (did not know how else to position the canvas behind the image).